src/test/java/uk/gov/justice/digital/delius/service/CourtReportServiceTest.java

```diff
@@ -21,22 +18,18 @@
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
-@RunWith(SpringRunner.class)
-@Import({CourtReportService.class, CourtReportTransformer.class, CourtAppearanceTransformer.class, CourtTransformer.class})
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
 public class CourtReportServiceTest {
-    @Autowired
     private CourtReportService courtReportService;
 
-    @MockBean
+    @Mock
     private CourtReportRepository courtReportRepository;
 
-    @MockBean
-    private LookupSupplier lookupSupplier;
 
-
-
-    @Before
+    @BeforeEach
     public void before() {
+        courtReportService = new CourtReportService(courtReportRepository);
```

Comment:
I know it wasn't for this change but can we import CourtReport into this class to remove the full package references below?